src/mlpack/methods/ann/layer/padding.hpp

```diff
@@ -41,11 +41,15 @@ class Padding
    * @param padWLeft Right padding width of the input.
    * @param padHTop Top padding height of the input.
    * @param padHBottom Bottom padding height of the input.
+   * @param aW number of extra zeros added to the right of input.
+   * @param aH number of extra zeros added to the top of input.
    */
   Padding(const size_t padWLeft = 0,
           const size_t padWRight = 0,
           const size_t padHTop = 0,
-          const size_t padHBottom = 0);
+          const size_t padHBottom = 0,
+          const size_t aW = 0,
```

Comment:
Not sure, but couldn't I use the exsisting `padWLeft` parameter to achive the same? Maybe I missed something?

Reply:
I used a separate variable just for clarity, however, we could just merge the value of aW into the existing variable, I'll make the changes in the next commit if using a separate variable is not necessary.

Reply:
The padWLeft variable was already being used in `output.submat()` so I left it that way, also it aW was used separately in the existing Pad() function. If merging is required then I'll make the changes in the next commit.

Reply:
In this case I would merge them.